src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +35,7 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user).map(x -> x.getAge()).filter(x -> x >=30 && x <= 45).isPresent();
```

Comment:
람다를 추가적으로 사용할 수 있습니다. ☺️

```suggestion
        return Optional.ofNullable(user).map(User::getAge).filter(x -> x >=30 && x <= 45).isPresent();
```

Reply:
`x >=30 && x <= 45`와 같은 조건을 의미있는 이름으로 메서드를 만들면 가독성이 더 좋을 것 같아요!